integration-cli/docker_cli_help_test.go

```diff
@@ -1,12 +1,13 @@
 package main
 
 import (
-	"os"
 	"os/exec"
 	"runtime"
 	"strings"
 	"testing"
 	"unicode"
+
+	"github.com/docker/docker/pkg/homedir"
 )
 
 func TestMainHelpWidth(t *testing.T) {
```

Comment:
@tiborvass I'm a little bit confused on this one.  If, when you run the tests, $HOME is `/` then https://github.com/docker/docker/blob/master/integration-cli/docker_cli_help_test.go#L84 should cause your test to fail because at some point it should find a `/` in your help text. The fact that it doesn't must mean that $HOME isn't really set to `/`.

However, now I realize that the `Contains()` in there is looking for too much. Instead of just looking for $HOME it should be looking for "$HOME  (ie. $HOME after a quote).  That should remove some of the potential false-negative results.  However, the check for $HOME being `/` is still needed because we don't want `/some/path` to become `~some/path`.